extras/manifests/prometheus-rules.yaml

```diff
@@ -16,6 +16,16 @@ spec:
     - expr: |
         sum(max(gluster_subvol_capacity_total_bytes{job="glusterd2-client"}) BY (volume, subvolume)) BY (volume)
       record: gluster:volume_capacity_total_bytes:sum
+  - name: exporter-absent
+    rules:
+    - alert: GlusterExporterDown
+      annotations:
+        message: GlusterExporter has disappeared from Prometheus target discovery.
```

Comment:
Is this message inline with k8s way of reporting absence of services?

Reply:
Yes, It is.
In fact, I have a feeling that same can be alerted via K8s alerts by a small tweak in our config.
Will look into it.

Reply:
Ack. If thats the case we rather should take that route instead of raising explicitly here.

Reply:
Looks like having explicit alert here is the safest bet. In case of duplicate alerts (only when integrated with k8s-mixin), alertmanager will take care of it, but in other cases we won't have duplicates and will work as expected.